### PR TITLE
Activity time slot view uses regime time slot ranges matching unlock list

### PR DESCRIPTION
**Pull request by iterative-it**

### CODE REVIEW

File: src/main/resources/migrations/common/V2023.09.30__v_activity_time_slot_range_fix.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+       activity_category.name
+FROM activity,
+     activity_schedule,
+     prison_regime,
+     scheduled_instance,
+     activity_category
+WHERE activity.activity_id = activity_schedule.activity_id
+  AND activity.prison_code::text = prison_regime.prison_code::text
+  AND activity_schedule.activity_schedule_id = scheduled_instance.activity_schedule_id
+  AND scheduled_instance.start_time < prison_regime.pm_start
```
Comment (iterative-it) [inline src/main/resources/migrations/common/V2023.09.30__v_activity_time_slot_range_fix.sql:16]: Previously `AND scheduled_instance.start_time = prison_regime.am_start`

File: src/main/resources/migrations/common/V2023.09.30__v_activity_time_slot_range_fix.sql [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+FROM activity,
+     activity_schedule,
+     prison_regime,
+     scheduled_instance,
+     activity_category
+WHERE activity.activity_id = activity_schedule.activity_id
+  AND activity.prison_code::text = prison_regime.prison_code::text
+  AND activity_schedule.activity_schedule_id = scheduled_instance.activity_schedule_id
+  AND scheduled_instance.start_time >= prison_regime.pm_start
+  AND scheduled_instance.start_time < prison_regime.ed_finish
```
Comment (iterative-it) [inline src/main/resources/migrations/common/V2023.09.30__v_activity_time_slot_range_fix.sql:34]: Previously `AND scheduled_instance.start_time = prison_regime.pm_start`

File: src/main/resources/migrations/common/V2023.09.30__v_activity_time_slot_range_fix.sql [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+       activity_category.name
+FROM activity,
+     activity_schedule,
+     prison_regime,
+     scheduled_instance,
+     activity_category
+WHERE activity.activity_id = activity_schedule.activity_id
+  AND activity.prison_code::text = prison_regime.prison_code::text
+  AND activity_schedule.activity_schedule_id = scheduled_instance.activity_schedule_id
+  AND scheduled_instance.start_time >= prison_regime.ed_start
```
Comment (iterative-it) [inline src/main/resources/migrations/common/V2023.09.30__v_activity_time_slot_range_fix.sql:51]: Previously `AND scheduled_instance.start_time = prison_regime.ed_start`

File: src/main/resources/migrations/common/V2023.09.30__v_activity_time_slot_range_fix.sql [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+FROM activity,
+     activity_schedule,
+     prison_regime,
+     scheduled_instance,
+     activity_category
+WHERE activity.activity_id = activity_schedule.activity_id
+  AND activity.prison_code::text = prison_regime.prison_code::text
+  AND activity_schedule.activity_schedule_id = scheduled_instance.activity_schedule_id
+  AND scheduled_instance.start_time >= prison_regime.pm_start
+  AND scheduled_instance.start_time < prison_regime.ed_start
```
Comment (iterative-it) [inline src/main/resources/migrations/common/V2023.09.30__v_activity_time_slot_range_fix.sql:34]: Previously `AND scheduled_instance.start_time = prison_regime.pm_start`

Comment (iterative-it) [inline src/main/resources/migrations/common/V2023.09.30__v_activity_time_slot_range_fix.sql:34]: Previously `AND scheduled_instance.start_time = prison_regime.pm_start`

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**